pkg/grpcserver/server.go

```diff
@@ -126,7 +141,9 @@ func (g *Server) Run(ctx context.Context) error {
 
 	err := g.server.Serve(g.listener)
 	if err != nil && !errors.Is(err, http.ErrServerClosed) {
-		g.logger.Error("grpc_server closed unexpected: %w", err.Error())
+		g.logger.WithFields(log.Fields{
```

Comment:
We don't want to have logs with high cardinality, and want just the general message of what happened in the log message and have the details in a separate field in case we want them